semgrep_metrics.atd

```diff
@@ -165,7 +192,10 @@ type misc = {
     ?numIgnored <ocaml mutable>: int option;
     ?ruleHashesWithFindings <ocaml mutable>: (string * int) list <json repr="object"> option;
     (* TODO: should be OSS | Pro, see semgrep_output_v1.atd engine_kind type *)
+    (* Still needed with engineConfig? *)
```

Comment:
Let's say something like "TODO deprecate this by 2/14/2023 and rely on pro_config"

Reply:
Sure.